README.md

```diff
@@ -1 +1,124 @@
-# OOP-school-library
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">🛰OOP school library🚀</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+- [📝 License](#license)
+
+# 📖 OOP school library <a name="about-project"></a>
+This repository includes Ruby files for an Object-Oriented Programming (OOP) school library app. The app is structured using classes to model students, teachers, and a basic library system.
+
+## Tech Stack <a name="tech-stack"></a>
+
+<ul>
+
+  <li><a href="https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/lessons/basic_syntax.md">Ruby</a></li>
+         
+</ul>
+
+## 🔑 Key Features <a name="key-features"></a>
+
+**Create Person class with:**
+  - Instance vars: @id, @name, @age
+  - Constructor with optional name, parent_permission
+  - Getters/setters for @name, @age
+  - Private of_age? method
+  - Public can_use_services? method
+  
+**Create Student subclass with:**
+  - Constructor adding @classroom
+  - play_hooky method
+  
+**Create Teacher subclass with:**
+  - Constructor adding @specialization
+  - Override can_use_services? to always return true
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby](https://www.ruby-lang.org/en/) installed and running. To get more information, read the [installation instructions](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md).
+- You should be able to use [IRB](https://en.wikipedia.org/wiki/Ruby_(programming_language)#Features).
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone https://github.com/prg-04/OOP-school-library.git
+  cd OOP-school-library
+```
+
+### Install
+
+Install project dependencies with:
+
+```
+  bundle install
+```
+
+### Usage <a name="usage"></a>
+
+To run the project, create a `main.rb` file and instantiate the class files in `main.rb` and execute the following command:
+
+```
+  ruby main.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Evans Karanja**
+
+- GitHub: [@githubhandle](https://github.com/prg-04)
+- Twitter: [@twitterhandle](https://twitter.com/Evans1425Sean)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/evanson-karanja-3549841b8/)
```

Comment:
- [ ] This link is currently broken, i kindly suggeset you provide the appropriate link to your linkedIn profile as it is unprofessional to leave broken links in your readme file :+1: :sparkles: